census/ui_app/EntryForm.jsx

```diff
@@ -177,7 +182,8 @@ const EntryForm = React.createClass({
           <div className="answer-wrapper">
             <div className="answer">
               <textarea name="details" rows="5"
-                        defaultValue={this.props.answers.details}></textarea>
+                        defaultValue={this.props.answers.details}
+                        {...readOnlyOpts}></textarea>
```

Comment:
That spread operator is super-useful. It means I can set the `readOnlyOpts` in a conditional manner somewhere outside of the return statement.